packages/runtime/src/components/builtin/Image/Image.tsx

```diff
@@ -17,6 +18,9 @@ import { cx } from '@emotion/css'
 import { useStyle } from '../../../runtimes/react/use-style'
 import { responsiveStyle, responsiveWidth } from '../../utils/responsive-style'
 import { useFile } from '../../../runtimes/react/hooks/makeswift-api'
+import { nextMajorVersion } from '../../../next/nextVersion'
```

Comment:
How come the filename is not following the naming conventions of other files?

Reply:
Fixed